tests/misc_testsuite/simd/cvt-from-uint.wast

```diff
@@ -0,0 +1,10 @@
+(module
+  (func (export "f32x4.convert_i32x4_u") (param v128) (result i32)
+    local.get 0
+    i32x4.extend_low_i16x8_u
+    f32x4.convert_i32x4_u
+    i8x16.extract_lane_s 10))
+
+(assert_return (invoke "f32x4.convert_i32x4_u" (v128.const i32x4 0x00000000 0x00000000 0x00000000 0x00000000))
+               (v128.const f32x4 0 0 0 0))
```

Comment:
I think we need to remove the extract lane part above?

Reply:
Yes, I also added more tests. Thanks.